ArduPlane/mode_rtl.cpp

```diff
@@ -108,11 +108,17 @@ void ModeRTL::navigate()
 
 // Switch to QRTL if enabled and within radius
 bool ModeRTL::switch_QRTL(bool check_loiter_target)
-{
-    if (!plane.quadplane.available() || (plane.quadplane.rtl_mode != QuadPlane::RTL_MODE::SWITCH_QRTL)) {
+{ 
+    if (!plane.quadplane.available() || ((plane.quadplane.rtl_mode != QuadPlane::RTL_MODE::SWITCH_QRTL) && (plane.quadplane.rtl_mode != QuadPlane::RTL_MODE::QRTL_ALWAYS))) {  
         return false;
     }
 
+   // if Q_RTL_MODE is QRTL always, then immediately switch to QRTL mode
+   if (plane.quadplane.rtl_mode == QuadPlane::RTL_MODE::QRTL_ALWAYS) {
+       plane.set_mode(plane.mode_qrtl, ModeReason:: QRTL_INSTEAD_OF_RTL);
```

Comment:
```suggestion
       plane.set_mode(plane.mode_qrtl, ModeReason::QRTL_INSTEAD_OF_RTL);
```
other uses do not include that whitespace

Reply:
thanks